llm_on_ray/common/tokenizer/empty_tokenizer.py

```diff
@@ -1,4 +1,4 @@
-from .tokenizer import Tokenizer
+from llm_on_ray.common.tokenizer.tokenizer import Tokenizer
```

Comment:
same, try to reduce it to this style by importing module rather than submodule
```suggestion
from llm_on_ray.common.tokenizer import Tokenizer
```